src/components/form/NewTaskForm.tsx

```diff
@@ -232,6 +241,19 @@ export const NewTaskForm = component$<NewTaskForm>(
 							<Button type='submit'>{t('ACTION_INSERT')}</Button>
 						</div>
 						<div class='text-xs text-darkgray-500'>{t('LEGEND_REQUIRED')}</div>
+						{isAdmin ? (
```

Comment:
```suggestion
						{isAdmin.value ? (
```

Altrimenti è sempre true, `isAdmin` è il signal